src/Microsoft.ML/Models/OnnxConverter.cs

```diff
@@ -10,9 +10,50 @@ namespace Microsoft.ML.Models
     public sealed partial class OnnxConverter
     {
         /// <summary>
-        /// Converts the model to ONNX format.
+        ///
```

Comment:
It might be worth providing some context on ONNX (at least linking to the ONNX website) and explaining why this is valuable. Similarly for Windows ML.